Makefile

```diff
@@ -263,6 +263,16 @@ sqs-image: sqs-linux
 	docker build -t $(IMAGE_PREFIX)aws-sqs-gateway:$(IMAGE_TAG) -f ./gateways/community/aws-sqs/Dockerfile .
 	@if [ "$(DOCKER_PUSH)" = "true" ] ; then  docker push $(IMAGE_PREFIX)aws-sqs-gateway:$(IMAGE_TAG) ; fi
 
+slack:
```

Comment:
This is not the problem of this PR, but how about putting Makefile in each gateway and include them by something like `include gateways/community/slack/Makefile` in the main `Makefile`?

Reply:
I will create a separate pr for this.